browser/src/app/TilesMiddleware.ts

```diff
@@ -1555,10 +1556,11 @@ class TileManager {
 	}
 
 	private static pixelCoordsToTwipTileBounds(coords: TileCoordData): number[] {
-		const x = coords.x * app.pixelsToTwips;
-		const y = coords.y * app.pixelsToTwips;
-		const width = app.tile.size.x;
-		const height = app.tile.size.y;
+		const pixelsToTwipsForTile = 15 / coords.scale;
```

Comment:
This is too arcane to leave here with no comment. Ideally it'd be a utility function and involve `app.pixelsToTwips`.

Reply:
We cannot use app.pixelsToTwips there. Added a comment.

Reply:
Let me note this, i am working on these things and will do it later.